doc/en/components/grids/_shared/row-pinning.md

```diff
@@ -235,11 +235,10 @@ grid.pinning = { rows: RowPinningPosition.Bottom };
 
 <!-- ComponentStart: Grid, TreeGrid, HierarchicalGrid -->
 ```tsx
-<{ComponentSelector} id="dataGrid" autoGenerate="true">
+<{ComponentSelector} id="dataGrid" ref={gridRef} autoGenerate={true}>
 </{ComponentSelector}>
 
-var grid = document.getElementById("dataGrid") as {ComponentSelector};
-grid.pinning = { rows: RowPinningPosition.Bottom };
+gridRef.current.pinning = { rows: RowPinningPosition.Bottom }
```

Comment:
Please set this as an input to the grid
`const pinning: IgrPinningConfig = { rows: RowPinningPosition.Bottom};`

Reply:
@tishko0 Perhaps you missed this?